### DISCUSSION TITLE
Feature Request: being able to reorder workspaces

### DISCUSSION THREAD

**Discussion by Tygo-van-den-Hurk** [zen-browser/desktop#2465] [Ideas]
Currently it's not possible to reorder workspaces. It'd be nice if there was some tab to like manage them. To reorder, or rename them etc.

**Comment by Fxxxlei** [comment]
It is possible to reorder and rename workspaces.

For reordering:
Right-click a workspace, click on the two arrows, drag&drop workspaces.

For renaming/changing icon:
Right-click a workspace, click on the three dots near a workspace, edit workspaces.

**Reply by Tygo-van-den-Hurk** [reply]
Yes you're right! The source from which I was getting my package was no longer being maintained. So it took me a little while to find out I was stuck at a6. I apologise for any time wasted, I'll close the discussion.
